app/controllers/games_controller.rb

```diff
@@ -168,9 +212,28 @@ def set_user_color
 
   # Never trust parameters from the scary internet, only allow the white list through.
   def game_params
-    params.require(:game).permit(:current_turn, :white_player_id, :is_blitz, :black_draw, :white_draw, :black_forfeit, :white_forfeit)
-
+    params.require(:game).permit(:current_turn, :white_player_id, :is_blitz, :black_draw, :white_draw, :black_forfeit, :white_forfeit, :has_started, :white_ready, :black_ready)
   end
   
+  def set_game
```

Comment:
I think these methods are in the application controller.. I think we can get rid of set_game and set_opponent_id